notebooks/basics/quantum-circuits.ipynb

```diff
@@ -343,7 +349,7 @@
     "\\end{aligned}\n",
     "$$\n",
     "\n",
-    "So, this circuit gives us a way to create the state $\\vert\\phi^+\\rangle$ if we run it on two qubits initialized to the $\\vert 00\\rangle$ state — and more generally it gives us a way to convert the standard basis to the Bell basis.\n",
+    "So, this circuit gives us a way to create the state $\\vert\\phi^+\\rangle$ if we run it on two qubits initialized to $\\vert 00\\rangle$. More generally, it gives us a way to convert the standard basis to the Bell basis.\n",
     "(The $-1$ phase factor on the last state, $-\\vert \\psi^-\\rangle$, could be eliminated if we wanted — by adding a controlled-Z gate at the beginning or a swap gate at the end, for instance.)"
```

Comment:
Comment on my own writing... seems like a comma works better than an m-dash here. Also should put Z in math mode:

"(The $-1$ phase factor on the last state, $-\vert\psi^-\rangle$, could be eliminated if we wanted, by adding a controlled-$Z$ gate at the beginning or a swap gate at the end, for instance.)"